### DISCUSSION TITLE
Request - masking shown achievements

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21123] [Other Feature Feedback, Questions, & Ideas] [answered]
Let's imagine a hypothetical a user who has three achievements: merging a PR without review, many stars on one of their repositories, and many pull requests merged.

The user considers the first to be shameful, and the last to be not so important, yet would still like to show the second on their account.

To remedy this, I propose that we may selectively show/hide unlocked achievements.

May such feature be considered?

**Comment by jfullstackdev** [comment, marked as answer]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
